WooCommerce/Classes/ViewRelated/Inbox/Inbox.swift

```diff
@@ -83,16 +61,33 @@ struct Inbox: View {
             }
             .toolbar {
                 ToolbarItem(placement: .navigationBarTrailing) {
-                    Button(action: {
-                        showingActionSheet = true
-                    }, label: {
-                        Image(uiImage: .moreImage)
-                            .renderingMode(.template)
-                    })
-                        .renderedIf(viewModel.syncState == .results)
+                    if viewModel.syncState == .results {
```

Comment:
Not a blocker: The toolbar button is visible when pulling to refresh the Inbox screen. Should we hide it while reloading?

https://github.com/user-attachments/assets/753f7224-875c-4bea-bce8-692fec5b458e

Reply:
Technically in this case there are still items in inbox, so I think it should be fine to let users dismiss the messages. I'll keep the behavior as-is for now if you don't mind.